schema/deploy/shipping/views.sql

```diff
@@ -2345,39 +2362,20 @@ grant select
 create or replace view shipping.scan_demographics_v2 as
 
     with hcov19_presence_absence as (
-        -- Collapse potentially multiple hCoV-19 results
-        select distinct on (sample_id)
-            sample.identifier as sample,
-            presence_absence_id,
-            case
-              when present is true then 'positive'
-              when present is false then 'negative'
-              when present is null then 'positive'
-            end as hcov19_result,
-            pa.created::date as hcov19_result_release_date
+        select
+          sample_id,
+          case
+            when present is true then 'positive'
+            when present is false then 'negative'
+            when present is null then 'positive'
+          end as hcov19_result,
+          presence_absence_id,
+          hcov19_result_release_date
         from
-            warehouse.presence_absence as pa
-            join warehouse.target using (target_id)
-            join warehouse.organism using (organism_id)
-            join warehouse.sample using (sample_id)
+          shipping.hcov19_presence_absence_result_v1 as pa
```

Comment:
Note to self: rename this alias to `hcov19_pa` to be consistent with other views and the commit message.

Reply:
Bump on the note to self :) Reading your commit message, I think hcov19_pa is a good rename! Thanks for documenting that.

Reply:
Thanks!